changelog_unreleased/api/pr-7466.md

```diff
@@ -0,0 +1,18 @@
+#### Change default value for `singleQuotes` to `true` ([#7466](https://github.com/prettier/prettier/pull/7466) by [@karlhorky](https://github.com/karlhorky))
+
+[Since version 0.0.1](https://github.com/prettier/prettier/commit/599b4311bb6be9204689a7725d37d6fdfca770aa), Prettier has an [option](https://prettier.io/docs/en/options.html#quotes) to use single quotes instead of double quotes.
+Since version 2.0, the default of this option changes from `false` to `true`.
+
+<!-- prettier-ignore -->
+```js
+// Input
+const foo = 'bar';
+// Prettier stable
+const foo = "bar";
+// Prettier master
+const foo = 'bar';
+```
+
+The JavaScript ecosystem has largely standardized on single quotes, with it being the most popular style seen in the wild.
```

Comment:
People are going to ask for sources on this.

Reply:
Hm, interesting perspective - I think they won't. I think that most JavaScript developers have had this experience by trolling through documentation and examples from all over the place.

But it's not about your opinion or my opinion, so maybe we can get some statistics together on this!

Some sources:

- GitHub usage
- public documentation of large JavaScript projects
- public blog posts
- developer surveys (@mvolkmann here: https://github.com/prettier/prettier/issues/4102#issuecomment-370476135)
- style guides

Reply:
But it's not about your opinion or my opinion, so maybe we can get some statistics together on this!

Some sources:

GitHub usage
public documentation of large JavaScript projects
public blog posts
developer surveys (@mvolkmann here: https://github.com/prettier/prettier/issues/4102#issuecomment-370476135)
style guides

Reply:
I will ask for some help in #4102.

Reply:
> I think that most JavaScript developers have already had this experience of single quotes being the norm by trolling through documentation

I write JS every day, and I honestly have no clue what’s the norm.

Reply:
Interesting take, maybe you just don't notice which quotes there are? Because I do, and I read about JavaScript from a ton of sources, from beginner level all the way up to spec level. I notice when I see double quotes, because it's so rare.

Reply:
@lydell Just finished a tour of many JavaScript library documentation pages, so go take a look over there: https://github.com/prettier/prettier/issues/4102#issuecomment-579912268

Reply:
Will continue my research (and get some other people to help too). Single quotes are almost everywhere, so collecting them isn't tough to do.

Reply:
Microsoft is really the only counter-example I can think of. They use doubled quotes.

The typescript source code: https://github.com/microsoft/TypeScript/blob/master/src/tsserver/server.ts

The typescript documentation: https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html

Reply:
@styfle Thanks! I've added this to the list.

Reply:
@karlhorky Maybe you could move the content of that comment into the original post because github auto-collapses comments in the middle so its unlikely someone will see that who is just finding this PR (like myself).

Reply:
Good idea. I moved it yesterday already actually, under the link text "my review of the ecosystem"